docs/_templates/footer.html

```diff
@@ -0,0 +1,14 @@
+{% extends "!footer.html" %}
+{% block extrafooter %}
+    <div id="dredd-hiring">
+        <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="Logo">
+        <div>
+            <h3>Dredd is hiring!</h3>
+            <p>Join the&nbsp;Apiary&nbsp;team and get&nbsp;paid for improving&nbsp;Dredd</p>
+        </div>
+        <a href="http://bit.ly/dredd-hiring" class="btn btn-neutral" target="_blank">
```

Comment:
This doesn't really matter since it's just a redirect but paranoid me with https. Feel free to ignore this suggestion :)

```suggestion
        <a href="https://bit.ly/dredd-hiring" class="btn btn-neutral" target="_blank">
```